specification.markdown

```diff
@@ -443,16 +443,18 @@ The range attribute is OPTIONAL. If the range attribute is not present then any
 
 ##### Expression of range
 
-The range attribute MUST only be used with EBML Elements that are either signed integer, unsigned integer, float, or date. The range expression may contain whitespace for readability but whitespace within a range expression MUST NOT convey meaning. The expression of the range MUST adhere to one of the following forms:
+The range attribute MUST only be used with EBML Elements that are either signed integer, unsigned integer, float, or date. The expression defines the upper, lower, exact or excluded value of the EBML Element and optionally an upper boundary value combined with a lower boundary. The range expression may contain whitespace for readability but whitespace within a range expression MUST NOT convey meaning.
 
-- x-y where x and y are integers or floats and y MUST be greater than x, meaning that the value MUST be greater than or equal to x and less than or equal to y. x MUST be less than y.
-- `>x where x` is an integer or float, meaning that the value MUST be greater than x.
-- `>=x where x` is an integer or float, meaning that the value MUST be greater than or equal to x.
-- `<x where x` is an integer or float, meaning that the value MUST be less than x.
-- `<=x where x` is an integer or float, meaning that the value MUST be less than or equal to x.
-- x where x is an integer or float, meaning that the value MUST be equal x.
+To set a fixed value for the range, the value is used as the attribute value. For example `1234` means the EBML element always has the value 1234. The value can be prefixed with `not` to indicate that the set value MUST not be used for that Element. For example `not 1234` means the Element can use all values of its type except 1234.
```

Comment:
Via rfc2119, `MUST NOT` should have the 'not' capitalized as well.

Reply:
should `always has the value 1234` be `MUST have a value of 1234`. The latter infers that the element is mandated to have that value whereas the former simply infers that it does have that value.

Reply:
I suggest that "with \`not\`" be "with \`not \`" (to include the space)

Reply:
> should `always has the value 1234` be `MUST have a value of 1234`. The latter infers that the element is mandated to have that value whereas the former simply infers that it does have that value.

This is an example so I don't think it should have anything normative.

Reply:
> I suggest that "with `not`" be "with `not `" (to include the space)

the backtick already produces a double quote, so it would end up like 
> The value can be prefixed "with ""not" to indicate that...